app/src/test/java/com/onelist/UserUnitTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.Test
 class UserUnitTest {
 
     @Test
-    fun userToString() {
+    fun `Given a user When the userID is 1 and the fistName is John Then the toString output should be 1 - John`() {
```

Comment:
Same as before, good habit to use Given When When to name unit tests. https://www.javacodegeeks.com/2017/05/importance-given-unit-tests-tdd.html